README.md

```diff
@@ -62,70 +100,66 @@ using       ton = unlib::kilo<kilogram>;       // humans don't use megagram
 A quantity combines a [unit](#units) (_mass_), a [scale](#scales) (_kilo_), a specific [value type](#values) (_double_), and an optional [tag](#tags) (_reactive_): 
 
 ```c++
-using kilogram = unlib::quantity<unlib::mass,unlib::kilo,double>. 
+using kilogram = unlib::quantity<unlib::mass,unlib::kilo,double>;
+using     watt = div_quantity_t<joule<V>,second<V>>;
 ```
 
-_(Note: All of unlib's predefined quantity types, including _kilogram_, are templates that take the value type – here **`double`** – as a template parameter.)_ 
+_(Note: All of unlib's predefined quantity types, including _kilogram_, are templates that take the value type – here fixed to **`double`** – as a template parameter.)_ 
 
-A quantity can be created from a value type explicitly: 
-
-```c++
-kilogram kg{42};
-```
-
-It cannot be implicitly created from a value type, though: 
+A quantity can be created from a value type explicitly. It cannot be implicitly created from a value type, though: 
 
 ```c++
 void f(unlib::quantity<unlib::mass,unlib::kilo,double>);
 
-f(kg);           // fine
+kilogram kg{42}; // fine
+f(kg);           // fine, too
 f(42);           // won't compile
 f(kilogram{42}); // also fine
 
 using namespace unlib::literals; // needed for unit suffixes
-f(42_kg);        // fine, too
+f(42_kg);        // also works
 ```
 
 A quantity also cannot be used where its value type is needed. If you need to pass a quantity to an API that expects one of the built-in value types instead, you have to explicitly convert it:
 
 ```c++
 void f(double kg);
 
-f(some_mass.get());                             // returns some_mass' underlying value, without regards for scaling
-f(some_mass.get_scaled<unlib::kilo_scaling>()); // will return kg independent of some_mass' scale
+f(some_mass.get());                             // returns underlying value, whatever scaling
+f(some_mass.get_scaled<unlib::kilo_scaling>()); // will always return kg 
```

Comment:
same mistake again, this will actually return megagram.

looking at this example and the problem that `unlib::quantity<unlib::mass,unlib::kilo,double>` is actually "Megagram", I'm not convinced that making kilogram the "unscaled" base unit is a good choice.

It is unfortunate that the SI unit system has a scaled base unit, but we can't change that.

Still, I think it might be better for unlib and the sanity of its users to:
- use _gram_ for the mass dimension exponent
- define the base unit kilogram as `template <typename V> using kilogram = quantity<mass              , kilo_scaling, V>;
- define all derived units in terms of the SI base units (or other derived units)

Reply:
OK, you are right on this. I will make `gram` the unscaled mass quantity again. Since all units are now defined in terms of quantities, using `kilogram` in the definition of, e.g. Newton, will make sure the derivations are correct nevertheless.


However... You just called `quantity<mass , kilo_scaling, V>` a "unit" and we just had this in verbal discussion, too. If so, then maybe "quantity" really _is_ indeed a misnomer? 
* sigh *